test/test.mk

```diff
@@ -13,9 +13,7 @@ GIT_PATH            = $(BR2_EXTERNAL_INFIX_PATH)
 endif
 GIT_VERSION         = $(shell git -C $(GIT_PATH) describe --dirty --always --tags)
 
-IMAGE ?= infix
-TOPOLOGY-DIR ?= $(test-dir)/virt/quad
-
+TOPOLOGY-DIR := $(test-dir)/virt/quad
```

Comment:
Since this does not need to be changed anymore, maybe we can remove it and just have...

```make
mode-qeneth := -q $(test-dir)/virt/quad
```

...below?

For future reference: I also like to put variables that can be overridden by the user in all-caps, but I will always `SNAKE_CASE` these, rather than `KEBAB-CASING` them. This makes it easier to pass them from a shell, since a dash is not a valid character in a  _shell_ variable, while it is in a _make_ variable.

Reply:
Yes, I'll fix that :)
Thnx